iib/workers/tasks/git_utils.py

```diff
@@ -84,13 +87,24 @@ def push_configs_to_git(
                 repo_configs_dir,
             )
 
-            if operator_packages:
+            if rm_operators:
+                # Remove operators from the Git repo if an RM operation is requested
+                for operator_package in set(rm_operators):
+                    if operator_package:  # We don't want to remove the whole /configs directory
```

Comment:
Consider changing lines 93-94 to 
```
operator_dir = os.path.join(repo_configs_dir, operator_package)
if os.path.isdir(operator_dir):
    try
    ...
```

Reply:
But in this case if `operator_package` is an empty string the `os.path.isdir(operator_dir)` will return True and we'll remove the `/configs`, no? The idea here is to check if `operator_package` is not empty so it can be safely removed.

Or is there a better way to accomplish that?

Reply:
Example:

```
repo_configs_dir = "/tmp/configs"
operator_package = ""

operator_dir = os.path.join(repo_configs_dir, operator_package)
os.path.isdir(operator_dir)  # will result in /tmp/configs
shutil.rmtree(operator_dir) # xxx
```

Reply:
@JAVGan In case the `operator_package` dir exists but it empty, running shutil.rmtree on `/configs/operator_package` would only remove the `operator_package` directory and not the parent directory. 

In your changes, on line 91, you are checking if `rm_operators` is not empty. So the operator_package variable will never be an empty string, no?

Reply:
Hmm ok, makes sense!

I'll patch it then as you suggested

Reply:
Patched, PTAL again